Validation/RecoTrack/src/SiStripTrackingRecHitsValid.cc

```diff
@@ -447,7 +447,7 @@ void SiStripTrackingRecHitsValid::beginJob(const edm::EventSetup& es){
   
 }
 
-void SiStripTrackingRecHitsValid::endJob() {
+inline void SiStripTrackingRecHitsValid::endJob() {
```

Comment:
I suspect that with the threaded DQM this method is in practice obsolete, and separate HARVESTING step should be used for this functionality (but probably it would be best to clean it up in a separate PR).

I also suspect that `inline` here serves no purpose (method is virtual and called by the framework, and anyway inline should be used in declarations rather than definitions).